packages/sdk/src/handlers/settings.ts

```diff
@@ -1,15 +1,29 @@
 /**
  * Settings handler
  */
-
-import { ItemsHandler } from '../base/items';
+import { OneItem, PartialItem, QueryOne } from '../items';
 import { ITransport } from '../transport';
 import { SettingType, DefaultType } from '../types';
 
 export type SettingItem<T = DefaultType> = SettingType & T;
 
-export class SettingsHandler<T = DefaultType> extends ItemsHandler<SettingItem<T>> {
+export class SettingsHandler<T = DefaultType> {
```

Comment:
Settings is a regular singleton, we might be able to extend the regular singleton handler here?